### PR TITLE
Add method to get elapsed time

### PR DESCRIPTION
**Pull request by KotaTakahashi9320**
# Summary
Add method to get elapsed time
Related to https://github.com/acompany-develop/QuickMPC/pull/62
# Purpose
To be able to get the elapsed time from the client
# Contents
gRPC protobuf update
Add method to get elapsed time
Add get_elapsed_time.py
# Testing Methods Performed
Run demo/unit_demo/get_elapsed_time.py

### CODE REVIEW

File: quickmpc/proto/libc_to_manage.proto [hunk 1]
```diff
@@ -167,3 +168,13 @@ message GetDataListResponse {
...
+
+message GetElapsedTimeRequest {
+    string job_uuid = 1;
+    string token = 2;
+}
+
+message GetElapsedTimeResponse {
+    bool is_ok = 1;
+    string result = 2;
+}
```
Comment (arukuka) [inline quickmpc/proto/libc_to_manage.proto]: Please add newline at EOF if you don't mind.

**Review by arukuka — CHANGES_REQUESTED** [review]

File: demo/unit_demo/get_elapsed_time.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    qmpc: QMPC = QMPC(
+        ["http://localhost:9001",
+         "http://localhost:9002",
+         "http://localhost:9003"]
+    )
+
+    # job_uuidを指定して計算結果を取得する
+    job_uuid: str = "10b6bfda-8776-4d8c-ad9a-b40fa277ff37"
+    res = qmpc.get_elapsed_time(job_uuid)
+    print(res)
```
Comment (KotaTakahashi9320) [inline demo/unit_demo/get_elapsed_time.py]:
```suggestion
```

**Comment by KotaTakahashi9320** [comment]
@arukuka @come2ry @mdonaka 
Please check the correction.

**Review by mdonaka — APPROVED** [review]

**Review by arukuka — APPROVED** [review]
Thank you!

**Pull request merged**